projects/demo/src/modules/components/items-with-more/examples/1/index.ts

```diff
@@ -13,15 +13,7 @@ import {TuiChip, TuiItemsWithMore} from '@taiga-ui/kit';
     changeDetection,
 })
 export default class Page {
-    protected readonly items = [
-        'John Cleese',
-        'Eric Idle',
-        'Graham Chapman',
-        'Michael Palin',
-        'Terry Gilliam',
-        'Terry Jones',
-    ];
-
+    protected readonly items = inject<readonly string[]>('Pythons' as any);
```

Comment:
nit: what about token with a factory for `Pythons`? to avoid "as any"

Reply:
Then we would need to import it, I made it so it's specifically easier to open on StackBlitz. We can use token instead of string, but then we would need an import, if you can – you can add it later, I'm not really sure how to do it properly.